bazel/repository_locations.bzl

```diff
@@ -1149,8 +1149,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WebAssembly for Proxies (C++ host implementation)",
         project_desc = "WebAssembly for Proxies (C++ host implementation)",
         project_url = "https://github.com/proxy-wasm/proxy-wasm-cpp-host",
-        version = "2cf68d6c00b4141ecf2afb5e5d51ebd21a27ed43",
-        sha256 = "6314d3e36e8aa9c4e792772b66f773fcfd58affcd4fd437c6d3dc540165e7fc6",
+        version = "4ddbed3c8c8c1aa76db8157e22ae3be676e76ac1",
```

Comment:
Wastime was updated to v0.39.1 in https://github.com/proxy-wasm/proxy-wasm-cpp-host/commit/9387a5b77418e41bbf2b0909c68c14db42a1e743.

Reply:
i added the commit from #22469 as it was getting `wasm_config` errors in the build without this

ive kept that commit but reverted the `proxy-wasm-cpp-host` to the one from the 0.39.1 update - lets see ..

Reply:
great seems to have worked - thanks @PiotrSikora